common-libs/user-management/src/TrueCallerService.ts

```diff
@@ -19,6 +19,7 @@ const { TRUECALLER_TOKEN_DEFAULT_EXPIRY_BUFFER } = process.env
 @profile()
 export class TrueCallerService {
   private readonly trueCallerUrl
+  private trueCallerPublicKeyManager: TrueCallerPublicKeyManager
 
   constructor() {
```

Comment:
please inject all dependencies with constructor.  
In case we need to change your implementation on some custom, how we can do this?